config.schema.yaml

```diff
@@ -96,3 +96,32 @@ jira_accounts:
         required: true
         secret: false
         default: "https://stfc.atlassian.net"
+
+icinga_accounts:
```

Comment:
We'll need to look at how we're deploying stackstorm using aquilon to add this config in before this gets merged